src/mode/aiy/mod.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @fileoverview AIY modifications.
+ *
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author fstanis@google.com (Filip Stanis)
+ */
+goog.provide('cwc.mode.aiy.Mod');
+
+goog.require('cwc.mode.aiy.Connection');
+goog.require('cwc.mode.aiy.Toolbar');
+goog.require('cwc.mode.aiy.Editor');
+goog.require('cwc.mode.aiy.Layout');
+goog.require('cwc.mode.default.Mod');
+goog.require('cwc.mode.aiy.Runner');
+goog.require('cwc.ui.Terminal');
```

Comment:
There is no Terminal instance any more there is only an global console.
The console should be only used for console typical stuff and not for specific modification, so if you need an terminal we need maybe to implement an global terminal emulator or something like this.